sides/thai-chile-brussels-sprouts.md

```diff
@@ -0,0 +1,42 @@
+# Thai chile Brussels sprouts
+
+Inspired by Austin's [East Side King](https://www.keyingredient.com/recipes/2265695032/fried-brussel-sprouts-east-side-king/), these Brussels sprouts are tangy, sweet, spicy, perfect.
+
+
+## Ingredients
+
+- 2 cup jasmine rice, dry
+- 1 cup sweet chili sauce, preferably Mae Ploy brand
+- 1 cup distilled white vinegar
+- 1 clove garlic, minced
+- 4 Thai chiles, minced
+- 2 tablespoons vegetable oil
+- 1 pound brussels sprouts, quartered
+- 1/2 cup red cabbage, thinly sliced
+- 1/2 cup green cabbage, thinly sliced
+- 1/4 cup alfalfa sprouts
+- 1/4 cup sweet onion, thinly sliced
+- 1 large jalapeño pepper, thinly sliced
+- salt, to taste
+- 1/8 cup fresh mint leaves, torn
+- 1/8 cup fresh cilantro leaves, torn
+- 1/8 cup fresh basil leaves, torn
+
+
+## Instructions
+
+1. Wash and start cooking the 2 cups of rice.
+
+2. Place the 1 cup chili sauce, 1 cup vinegar, 1 clove minced garlic, and 4 minced Thai chiles in a small mixing bowl. Mix well and set aside.
+
+3. Heat the 2 tablespoons oil in a large skillet over high heat.
+
+4. Once the oil is shimmering, add the 1 pound quartered brussels sprouts and cook them until the cores of the sprouts are approaching golden brown and the outside starts to char and get crispy, about 2 - 3 minutes.
```

Comment:
I totally forgot that the recipe cooks in a pan. I've usually roasted them in the oven like in this recipe: https://walkingonsunshinerecipes.com/how-to-make-easy-and-delicious-roasted/?utm_campaign=yummly&utm_medium=yummly&utm_source=yummly#mv-creation-699-jtr